Source/Scene/parseKTX2.js

```diff
@@ -0,0 +1,225 @@
+import arraySlice from "../Core/arraySlice.js";
+import defined from "../Core/defined.js";
+import PixelFormat from "../Core/PixelFormat.js";
+import RuntimeError from "../Core/RuntimeError.js";
+import VulkanConstants from "../Core/VulkanConstants.js";
+import PixelDatatype from "../Renderer/PixelDatatype.js";
+import { read } from "../ThirdParty/Workers/ktx-parse.modern.js";
+
+var faceOrder = [
+  "positiveX",
+  "negativeX",
+  "positiveY",
+  "negativeY",
+  "positiveZ",
+  "negativeZ",
+];
+
+// Flags
+var colorModelETC1S = 163;
+var colorModelUASTC = 166;
```

Comment:
From https://github.com/KhronosGroup/glTF/tree/master/extensions/2.0/Khronos/KHR_texture_basisu#ktx-v2-images-with-basis-universal-supercompression

> To cover a broad range of use cases, this extension allows two Basis Universal formats: ETC1S with BasisLZ and UASTC with optional Zstandard compression.

Have you tried loading UASTC with Zstandard compression? Does the basis transcoder handle that?